src/components/pages/Chat/index.tsx

```diff
@@ -52,6 +54,16 @@ const UnstyledChat = ({ className }: Props) => {
         }
     }, [sessionAccount, navigate])
 
+    const erc20RequestDelegatedJoin = useEJPRequestDelegatedJoin()
+    const [isMount, setIsMount] = useState(false)
+    useEffect(() => {
+        if (isMount) {
+            return
+        }
+        setIsMount(true)
+        erc20RequestDelegatedJoin({})
+    })
```

Comment:
No deps for `useEffect`? i.e. `[isMount, erc20RequestDelegatedJoin]`?

If you want to do something on mount only then

```jsx
const erc20RequestDelegatedJoinRef = useRef(erc20RequestDelegatedJoin)

useEffect(() => {
    erc20RequestDelegatedJoinRef.current({})
}, [])
```